src/gamma.jl

```diff
@@ -926,3 +926,22 @@ function logabsbinomial(n::T, k::T) where {T<:Integer}
     end
 end
 logabsbinomial(n::Integer, k::Integer) = logabsbinomial(promote(n, k)...)
+
+
+# this trickery is needed while the deprecated method in Base exists
```

Comment:
What deprecation?  As far as I can see there is no suggestion of deprecating `binomial` in Base, and `binomial` in Base has only `Integer` methods.

Reply:
Deleted all of that. See commit c89743a.